ibis/backends/pandas/aggcontext.py

```diff
@@ -266,29 +266,37 @@ def agg(self, grouped_data, function, *args, **kwargs):
 
 def wrap_for_apply(
     function: Callable,
-    args: Optional[Tuple[Any]],
-    kwargs: Optional[Dict[Any, Any]],
+    args: Optional[Tuple[Any, ...]] = None,
+    kwargs: Optional[Dict[str, Any]] = None,
```

Comment:
I'm sure there is a reason for this, but why do we need to set a default value for the annotations? If there wasn't a default value, I assume all the calls provide those values. It's not obvious how this helps.

Reply:
The default value was not added for annotations but because there is a use of this function were those parameters were not provided. I was tempted of removing the Optional annotation but later find that call. Here you can see the call https://github.com/ibis-project/ibis/blob/18e967cac961285b05d8df560f40148bac1a2571/ibis/backends/pandas/aggcontext.py#L590-L593 

If you're ok, I can remove the Optional part, the default values and change the call to provide an empty tuple and dict instead.